concent_api/core/tests/utils.py

```diff
@@ -157,7 +157,7 @@ def _get_deserialized_task_to_compute(
             elif isinstance(deadline, str):
                 compute_task_def['deadline'] = self._parse_iso_date_to_timestamp(deadline)
             else:
-                compute_task_def['deadline'] = self._parse_iso_date_to_timestamp(self._get_timestamp_string())
+                compute_task_def['deadline'] = self._parse_iso_date_to_timestamp(self._get_timestamp_string()) + 10
```

Comment:
It isn't. It is a default value for deadline, because if sometime message is created without `deadline` parameter it has same value as timestamp so better to give time what is some kind in future than not :)